biome.json

```diff
@@ -88,7 +88,8 @@
       "a11y": {
         "useAnchorContent": "off",
         "noSvgWithoutTitle": "off",
-        "noNoninteractiveTabindex": "off"
+        "noNoninteractiveTabindex": "off",
+        "noNoninteractiveElementToInteractiveRole": "off"
```

Comment:
Maybe we shouldn't add this as global?

Moreover, let's add role button to the Carousel.Item ?

Reply:
> Maybe we shouldn't add this as global?

Makes sense

> Moreover, let's add role button to the Carousel.Item ?

It should has tabpanel role